pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -90,7 +90,7 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 			return err
 		}
 		if listener == nil {
-			s.logger.Info("Creating load balancer listener", "name", lbPortObjectsName)
+			s.logger.Info("Creating load balancer listener", "name", lbPortObjectsName, "lbid", lb.ID)
```

Comment:
```suggestion
			s.logger.Info("Creating load balancer listener", "name", lbPortObjectsName, "lb-id", lb.ID))
```
I think `-` is the convention. Wanted to open an issue soon for that, just didn't get to it.